phi3-finetune/README.md

```diff
@@ -0,0 +1,58 @@
+# Phi-3 Model Fine-tuning Demo
+
+This demo will show how to use ACPT (Azure Container for PyTorch) along with accelerators such as onnxruntime training (through ORTModule) and DeepSpeed to fine-tune Phi-3 model.
+
+## Background
+
+[Phi-3](https://www.microsoft.com/en-us/research/blog/phi-3-the-surprising-power-of-small-language-models/)  is 2.7 billion-parameter language model with nex-t word prediction objective. It has been trained using mixture of Synthetic and Web datasets.
+
+## Set up
+
+### AzureML
+The easiest option to run the demo will be using AzureML as the environment details are already included, there is another option to run directly on the machine which is provided later. For AzureML, please complete the following prerequisites:
+
+#### Local environment
+Set up your local environment with az-cli and azureml dependency for script submission:
+
+```
+az-cli && az login
+pip install azure-ai-ml azure-identity
+```
+
+#### AzureML Workspace
+- An AzureML workspace is required to run this demo. Download the config.json file ([How to get config.json file from Azure Portal](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-configure-environment#workspace)) for your workspace. Make sure to put this config file in this folder and name it ws_config.json.
+- The workspace should have a gpu cluster. This demo was tested with GPU cluster of SKU [Standard_ND40rs_v2](https://docs.microsoft.com/en-us/azure/virtual-machines/ndv2-series). See this document for [creating gpu cluster](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster?tabs=python). We do not recommend running this demo on `NC` series VMs which uses old architecture (K80).
```

Comment:
AML only allows A100 to train phi-3 models. We need to align with AML and changed the statement here.